### PR TITLE
Add support for Aquaero flow sensor 1

### PR DESCRIPTION
**Pull request by aleksamagicka**
Offset found and tested in [#50](https://github.com/aleksamagicka/aquacomputer_d5next-hwmon/issues/50#issuecomment-1332529792).

### CODE REVIEW

**Review by leoratte — CHANGES_REQUESTED** [review]
Good addition, but I think you can reuse some of the code for the quadro flow sensor.

File: aquacomputer_d5next.c [hunk 1]
```diff
@@ -113,11 +113,12 @@ static u8 aquastreamxt_secondary_ctrl_report[] = {
 /* Register offsets for Aquaero fan controllers */
 #define AQUAERO_SERIAL_START		0x07
 #define AQUAERO_FIRMWARE_VERSION	0x0B
-#define AQUAERO_NUM_FANS		4
+#define AQUAERO_NUM_FANS		5
```
[the remark this answers is not in the document]
Comment (aleksamagicka) [reply aquacomputer_d5next.c:116]: True, thanks. Remembered while I was implementing the Aquastream Ultimate support right after.

File: aquacomputer_d5next.c [hunk 2]
```diff
@@ -1526,6 +1536,9 @@ static int aqc_raw_event(struct hid_device *hdev, struct hid_report *report, u8
 
 	/* Special-case sensor readings */
 	switch (priv->kind) {
+	case aquaero:
+		priv->speed_input[4] = get_unaligned_be16(data + AQUAERO_FIRST_FLOW_SENSOR);
+		break;
```
[the remark this answers is not in the document]
Comment (aleksamagicka) [reply aquacomputer_d5next.c:1541]: Will do. Here we support only the first Aquaero flow sensor so it fits in nicely.

**Review by leoratte — APPROVED** [review]
Looks good now

**Comment by aleksamagicka** [comment]
Thanks!

**Pull request merged**